.gitpod.yml

```diff
@@ -15,6 +15,19 @@ tasks:
       cd "$GITPOD_REPO_ROOT"
       cp .bashrc.d/100-firefox-dev ~/.bashrc.d/
       source ~/.bashrc.d/100-firefox-dev
+      
+      # Try server: if both TRY_USERNAME and TRY_SSH_KEY environment variables are present,
+      # configure the credentials for pushing to try.
+      #
+      # (This is not required, but useful if you have Try access and want to push builds to try.)
```

Comment:
I think we should have a new section in the `README.md` file with the extra Gitpod configuration because we could have other envs (like for `moz-phab`).

Reply:
Indeed! I added the relevant documentation to the README – and also, for good measure, the env var for configuring `moz-phab` that I needed too.